selftests/unit/test_mux.py

```diff
@@ -369,7 +383,8 @@ def test_get_greedy_path(self):
         # path matches nothing
         self.assertEqual(self.params1.get('root', '', 999), 999)
 
-    @unittest.skipIf(not yaml_to_mux.MULTIPLEX_CAPABLE, "Not multiplex capable")
+    @unittest.skipUnless(plugin_available("avocado-framework-plugin-varianter-yaml-to-mux"),
+                         "yaml_to_mux plugin available")
```

Comment:
I suppose you meant `Plugin yaml_to_mux not available` here.

Reply:
True!

Reply:
I'm changing them all:

```diff
diff --git a/selftests/unit/test_mux.py b/selftests/unit/test_mux.py
index 15fb99bb..39aa5686 100644
--- a/selftests/unit/test_mux.py
+++ b/selftests/unit/test_mux.py
@@ -245,7 +245,7 @@ class TestMuxTree(unittest.TestCase):
 class TestMultiplex(unittest.TestCase):
 
     @unittest.skipUnless(plugin_available("avocado-framework-plugin-varianter-yaml-to-mux"),
-                         "yaml_to_mux plugin available")
+                         "Plugin avocado_varianter_yaml_to_mux not available")
     def setUp(self):
         self.mux_tree = yaml_to_mux.create_from_yaml(['/:' + PATH_PREFIX +
                                                       'selftests/.data/mux-selftest.'
@@ -310,14 +310,14 @@ class TestAvocadoParams(unittest.TestCase):
                                                 ['/ch1/*', '/ch0/*'])
 
     @unittest.skipUnless(plugin_available("avocado-framework-plugin-varianter-yaml-to-mux"),
-                         "yaml_to_mux plugin available")
+                         "Plugin avocado_varianter_yaml_to_mux not available")
     def test_pickle(self):
         params = pickle.dumps(self.params1, 2)  # protocol == 2
         params = pickle.loads(params)
         self.assertEqual(self.params1, params)
 
     @unittest.skipUnless(plugin_available("avocado-framework-plugin-varianter-yaml-to-mux"),
-                         "yaml_to_mux plugin available")
+                         "Plugin avocado_varianter_yaml_to_mux not available")
     def test_basic(self):
         self.assertEqual(self.params1, self.params1)
         self.assertNotEqual(self.params1, self.params2)
@@ -327,7 +327,7 @@ class TestAvocadoParams(unittest.TestCase):
         self.assertEqual(15, sum([1 for _ in iteritems(self.params1)]))
 
     @unittest.skipUnless(plugin_available("avocado-framework-plugin-varianter-yaml-to-mux"),
-                         "yaml_to_mux plugin available")
+                         "Plugin avocado_varianter_yaml_to_mux not available")
     def test_unhashable(self):
         """ Verifies that unhashable arguments can be passed to params.get """
         self.assertEqual(self.params1.get("root", "/ch0/", ["foo"]), ["foo"])
@@ -336,7 +336,7 @@ class TestAvocadoParams(unittest.TestCase):
                                           ['bar']), 'unique1')
 
     @unittest.skipUnless(plugin_available("avocado-framework-plugin-varianter-yaml-to-mux"),
-                         "yaml_to_mux plugin available")
+                         "Plugin avocado_varianter_yaml_to_mux not available")
     def test_get_abs_path(self):
         # /ch0/ is not leaf thus it's not queryable
         self.assertEqual(self.params1.get('root', '/ch0/', 'bbb'), 'bbb')
@@ -359,7 +359,7 @@ class TestAvocadoParams(unittest.TestCase):
                                           'hhh'), 'hhh')
 
     @unittest.skipUnless(plugin_available("avocado-framework-plugin-varianter-yaml-to-mux"),
-                         "yaml_to_mux plugin available")
+                         "Plugin avocado_varianter_yaml_to_mux not available")
     def test_get_greedy_path(self):
         self.assertEqual(self.params1.get('unique1', '/*/*/*/ch0.1.1.1/',
                                           111), 'unique1')
@@ -384,7 +384,7 @@ class TestAvocadoParams(unittest.TestCase):
         self.assertEqual(self.params1.get('root', '', 999), 999)
 
     @unittest.skipUnless(plugin_available("avocado-framework-plugin-varianter-yaml-to-mux"),
-                         "yaml_to_mux plugin available")
+                         "Plugin avocado_varianter_yaml_to_mux not available")
     def test_get_rel_path(self):
         self.assertEqual(self.params1.get('root', default='iii'), 'root')
         self.assertEqual(self.params1.get('unique1', '*', 'jjj'), 'unique1')
```